api_tests/config.py

```diff
@@ -22,10 +22,11 @@ def parse():
 
     # Make sure config file has all required keys
     missing_key = verify_keys(config_dict, [
-      'url',
-      'access_token',
-      'snippet',
-      'user_id',
+        'url',
+        'access_token',
+        'snippet',
+        'user_id',
+        'invalid_snippet_key',
```

Comment:
How are we determining which snippet to delete? If the value is hardcoded, that's a collision risk too, in which case it should be refactored to use this config variable.